app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  private
+
+  def update_likes_counter
+    post.increment!(:likes_count)
+  end
```

Comment:
- [ ] Kindly apply the comment of after_save which I made in post.rb to this method also.